dockerfiles/base/scripts/base/commands/cmd_start.sh

```diff
@@ -165,13 +165,13 @@ cmd_start_check_postflight() {
 cmd_stop() {
   debug $FUNCNAME
   FORCE_STOP=false
-  if [[ "$@" == *"--force"* ]]; then
+  if [[ "$@" == *"--skip:graceful"* ]]; then
   	FORCE_STOP=true
   fi
 
   if [[ ${FORCE_STOP} = "false" ]]; then
-    info "Waiting for graceful stop of services..."
-    cmd_action "graceful-stop" "$@"
+    info "stop" "Stopping workspaces..."
+    cmd_action "graceful-stop" "$@" >> "${LOGS}" 2>&1 || true
```

Comment:
if graceful stop fail, it would mean that we continue the stop command? I'm not sure it's what we should do if it's the case
if graceful fail, user need to figure out what he wants to do. Either he performs a force or he tries again the graceful stop